README.md

```diff
@@ -101,6 +101,7 @@ CIP Editors meetings are public, recorded, and [published on Youtube](https://ww
 | 0110 | [Plutus v1 compatible script references](./CIP-0110) | Proposed |
 | 0114 | [CBOR Tags Registry](./CIP-0114) | Proposed |
 | 0115 | [CBOR tag definition - ED25519-BIP32 Keys](./CIP-0115) | Proposed |
+| 0122 | [Logical operations](./CIP-0122) | Proposed | 
```

Comment:
```suggestion
```
@kozross I appreciate the initiative in adding this, but I'd only thrown this change into a CIP PR as per https://github.com/cardano-foundation/CIPs/pull/624#issuecomment-2092353644 because such an old proposal was being suddenly merged & legitimised.  Generally it helps editorially if editors themselves can submit one PR to do all these updates every 2 weeks so we don't end up with merge conflicts across branches from different submitters.

Reply:
No problem, can remove.